wgpu/src/lib.rs

```diff
@@ -179,21 +182,22 @@ trait Context: Debug + Send + Sized + Sync {
     type PipelineLayoutId: Debug + Send + Sync + 'static;
```

Comment:
Let's move the definition of the context trait into the same place as the dyn context one, that way you only have to update it once.